common/common.go

```diff
@@ -295,3 +295,14 @@ const (
 	// Keep alive is 2x enforcement minimum to ensure network jitter does not introduce ENHANCE_YOUR_CALM errors
 	GRPCKeepAliveTime = 2 * GRPCKeepAliveEnforcementMinimum
 )
+
+// Security severity logging
+const (
+	SecurityField     = "security"
+	SecurityCWEField  = "cwe"
```

Comment:
Tiny nitpick: maybe capitalize this to make it just a tiny bit clearer at a glance that it's an acronym?